main.py

```diff
@@ -1,10 +1,42 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        users = json.loads(file.read())
```

Comment:
Consider adding error handling for JSON parsing to manage cases where the file might not be properly formatted.